### PR TITLE
Use official Vue extension in Gitpod

### PR DESCRIPTION
**Pull request by yardenshoham**
The previous one didn't work well

### CODE REVIEW

**Review by wxiaoguang — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

File: .gitpod.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ vscode:
     - golang.go
     - stylelint.vscode-stylelint
     - DavidAnson.vscode-markdownlint
-    - johnsoncodehk.volar
+    - vue.volar
```
Comment (silverwind) [inline .gitpod.yml]:
```suggestion
    - Vue.volar
```

https://marketplace.visualstudio.com/items?itemName=Vue.volar

image

**Review by silverwind — APPROVED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (yardenshoham) [reply .gitpod.yml]: In the Gitpod web IDE 
image

I checked and it works on your commit, LGTM

**Review by lunny — APPROVED** [review]

**Review by delvh — APPROVED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (silverwind) [reply .gitpod.yml]: Yeah those are likely case-insensitive anyways, but I trust the vscode store more.

Comment (yardenshoham) [reply .gitpod.yml]: Just FYI Gitpod doesn't use that marketplace. https://www.gitpod.io/docs/references/ides-and-editors/vscode-extensions

**Pull request merged**